TW 517 Cycle 4 edits (#4090)
postmanlabs/postman-docs: src/pages/docs/publishing-your-api/custom-doc-domains.md

[![edit view for team](https://assets.postman.com/postman-docs/edit-team-profile-a.jpg)](https://assets.postman.com/postman-docs/edit-team-profile-a.jpg)

> In the __Custom Domains__ section, there is a list of domains indicated with verification status. Select __Add custom domain__.

Author: ```suggestion
The __Custom Domains__ section contains a list of domains with the verification status of each. Select __Add custom domain__.
```
